lib/output.js

```diff
@@ -115,17 +115,22 @@ module.exports = {
       }
     }
 
-    let stepLine = step.toCliStyled()
-    if (step.metaStep && outputLevel >= 1) {
-      // this.stepShift += 2;
-      stepLine = colors.dim(truncate(stepLine, this.spaceShift))
-    }
-    if (step.comment) {
-      stepLine += colors.grey(step.comment.split('\n').join('\n' + ' '.repeat(4)))
-    }
+    try {
```

Comment:
maybe better would be:

```js
if (step.constructor?.name == Step) {
  // ...
} elseif (typeof step == 'string') {
} else {
   // incorrect step
}
```

try catch is not the best approach here if we know where error can happen and how to avoid it